src/utils.ts

```diff
@@ -1,16 +1,29 @@
 import * as editorconfig from 'editorconfig';
-import {EditorSettings} from './interfaces/editorSettings';
-import {window, workspace, TextDocument, TextEditor} from 'vscode';
+import {
+	EditorSettings
+} from './interfaces/editorSettings';
+import {
+	window,
+	TextDocument,
+	TextEditor
+} from 'vscode';
 
 export class Utils {
 
 	/**
 	 * Convert .editorconfig values to vscode editor options
 	 */
-	public static fromEditorConfig(config: editorconfig.knownProps, defaults: EditorSettings): any {
+	public static fromEditorConfig(
+		config: editorconfig.knownProps,
+		defaults: EditorSettings
+	) {
```

Comment:
Actually I like the old style more to be honest. But I see the benefit for diffs. It's easier to see what changed if one type changes or an extra parameter was added. The code now looks a little bit cluttered to me.